src/Util/Search.php

```diff
@@ -807,32 +737,21 @@ private function splitQueryTerms(string $queryTerm): array
      *
      * @param string $mustNotQueryTerm Query term that needs to be searched upon.
      *
-     * @return Query\BoolQuery
+     * @return Query\MultiMatch
      */
-    private function getMustNotIncludeTermsQuery(string $mustNotQueryTerm): Query\BoolQuery
+    private function getMustNotIncludeTermsQuery(string $mustNotQueryTerm): Query\MultiMatch
     {
-        $mustNotBoolQuery = new Query\BoolQuery();
-
         $mustNotMultiMatch = new Query\MultiMatch();
         $mustNotMultiMatch->setFields(
```

Comment:
Don't we need the same cross_field type here?

Reply:
No. We don't need it, so that the given term does not match in any field